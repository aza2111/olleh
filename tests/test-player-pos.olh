//test-player-pos: positive test for player functionality

//Player player = fresh Player{};
//player.turn = true;
//print player.turn;

Player player = fresh Player{turn -> true, letters -> ['a','b']};
print player.turn;
print player.letters;

print (fresh Player{turn->true}).turn;
player.score = 89;
print player.score;
print player.guessedWords;

player.guessedWords = {"a"->1, "blah"->1};
print player.guessedWords;

player.guessedWords.set("asdf", 1);
print player.guessedWords;

player.guessedWords = fresh map(String);
player.guessedWords.set("asdf", 1);
print player.guessedWords;
