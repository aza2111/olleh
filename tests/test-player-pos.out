true
['a', 'b']
true
89
{}
{"blah" -> 1, "a" -> 1}
